diff --git a/boost/boost_1_66_0/libs/filesystem/src/operations.cpp b/boost/boost_1_66_0/libs/filesystem/src/operations.cpp
index f59cbae..ea9c6cf 100644
--- a/libs/filesystem/src/operations.cpp
+++ b/libs/filesystem/src/operations.cpp
@@ -1541,6 +1541,7 @@ namespace detail
 #   if defined(AT_FDCWD) && defined(AT_SYMLINK_NOFOLLOW) \
       && !(defined(__SUNPRO_CC) || defined(__sun) || defined(sun)) \
       && !(defined(linux) || defined(__linux) || defined(__linux__)) \
+      && !(defined(__QNX__) || defined(__QNXNTO__)) \
       && !(defined(__MAC_OS_X_VERSION_MIN_REQUIRED) \
            && __MAC_OS_X_VERSION_MIN_REQUIRED < 101000) \
       && !(defined(__IPHONE_OS_VERSION_MIN_REQUIRED) \
